pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>4.32</version>
+        <version>4.53</version>
```

Comment:
The dependabot tried to bump the version, but the CI has failed https://github.com/jenkinsci/casdoor-auth-plugin/pull/5.
Have you tested this plugin can be built with 4.53?

Reply:
>Have you tested this plugin can be built with 4.53?
>
Yes i did, i ran `mvn clean verify` and `mvn clean test` and both builds succeeded